src/openrct2/FileClassifier.h

```diff
@@ -48,7 +48,7 @@ struct ClassifiedFileInfo
     uint32_t Version = 0;
 };
 
-#define FILE_TYPE_S4_CUTOFF 2
+constexpr uint8_t kFileTypeS4Cutoff = 2;
```

Comment:
Shouldn't it be `uint32_t`, since it's always only compared to `ClassifiedFileInfo::Version`, which is from that type